modules/ocf_desktop/manifests/stats.pp

```diff
@@ -24,6 +24,16 @@
       mode   => '0555',
       owner  => ocfstats,
       source => 'puppet:///modules/ocf_desktop/stats/update-delay.sh';
+      
+    '/opt/stats/update-flags.sh':
+      mode   => '0555',
+      owner  => ocfstats,
+      source => 'puppet:///modules/ocf_desktop/stats/update-flags.sh';
+      
+    '/opt/stats/staff-visibility.py':
+      mode   => '0555',
```

Comment:
`o+x` and `g+x` seems like a mistake here. Say i'm lazy and antisocial (which i am), and someone comes in fix their wordpress. Since im lazy and antisocial, I dont want to help. I notice `rrchan` is in the lab, studying hard for a midterm and has his visibility toggled off. I then `ssh` to your computer and toggle you on, and now you have to help fix some broken wordpress install and fail your midterm miserably.

The correct perms should be `544`.